dataland-frontend/tests/e2e/specs/eu-taxonomy-financials/DataIntegrity.ts

```diff
@@ -202,11 +203,6 @@ describeIf(
           testData.companyInformation,
           testData.t
         );
-        checkCreditInstitutionValues(testData.t, true, true);
-
-        checkCommonFields("AssetManagement", testData.t.eligibilityKpis!.AssetManagement);
-
-        checkInsuranceValues(testData.t);
```

Comment:
What is the reason that we don't want to do those checks here anymore?

Reply:
Marcin says he does not know a reason for it.

=> So we should investigate if we put this back here

Reply:
Testing a "fully filled eu taxo financial dataset" happens in DataUpload.ts now via the form.